gnosis/safe-react
# Add breadcrums to app sections

## What it solves
Resolves #2540

## How this PR fixes it
Uses latest version of `safe-react-components` which includes a Breadcrumbs component.
Adds this breadcrumb component to all sections of the app
Those which include subsections will show a second level in the breadcrumb with the sub section name.

## How to test it
Navigate to each of the different routes and check that Breadcrumbs are shown and behave as expected

## Screenshots
Some representative examples:
![image](https://user-images.githubusercontent.com/25051234/125941520-8ea0e781-40e6-4426-bd9c-e508b3dc94cf.png)
![image](https://user-images.githubusercontent.com/25051234/125941550-6e3237e6-6bdb-4a87-9e1f-c8c25501eaa6.png)
![image](https://user-images.githubusercontent.com/25051234/125941567-867ea15b-5e6e-49df-948c-ac23c0766b47.png)
